### PR TITLE
WIP: Kubernetes 1.3

### PR DESCRIPTION
**Pull request by ibuildthecloud**

### CODE REVIEW

File: templates/kubernetes/7/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.catalog:
+  name: Kubernetes
+  version: v1.3.0-rancher1
+  description: Rancher Kubernetes service
+  minimum_rancher_version: v1.1.0
```
Comment (alena1108) [inline templates/kubernetes/7/rancher-compose.yml]: Should this version be v1.1.2? or whatever official version pre-release would have

Comment (ibuildthecloud) [inline templates/kubernetes/7/rancher-compose.yml]: Correct, but i had weird issue with that.  I think it should be 1.2.0.  We won't merge until we sort that out.

File: templates/kubernetes/7/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        KUBERNETES_URL: http://master
+    image: rancher/kubernetes-agent:v0.3.2
+    privileged: true
+    volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+    links:
+        - kubernetes:master
+
+rancher-ingress-controller:
+    image: rancher/lb-controller:v0.2.0
```
Comment (alena1108) [inline templates/kubernetes/7/docker-compose.yml]: @ibuildthecloud v0.2.1...for now :)

**Comment by vincent99** [comment]
Moved to https://github.com/rancher/rancher-catalog/pull/181

**Pull request closed without merging**